pkg/rsyslog/Dockerfile.in

```diff
@@ -0,0 +1,20 @@
+# hadolint ignore=DL3006
+FROM GPTTOOLS_TAG as gpttools
+
+FROM alpine:3.8
+WORKDIR /
+COPY rsyslog@lists.adiscon.com-5a55e598.rsa.pub /etc/apk/keys/rsyslog@lists.adiscon.com-5a55e598.rsa.pub
+# hadolint ignore=DL3018
+RUN echo "http://alpine.rsyslog.com/3.7/stable" >> /etc/apk/repositories \
+            && apk --no-cache update \
+            && apk add --no-cache \
```

Comment:
You don't need apk --no-cache update here

Reply:
Will remove.